bin/integration_tests

```diff
@@ -71,9 +71,31 @@ run_python_tests(){
     nose2 --plugin nose2.plugins.junitxml --with-coverage --coverage-report xml -X -v -s test ${test_params}
 }
 
+run_csharp_tests(){
+  if [ $no_regen_client -eq 0 ]; then
+    announce "Generating C# Client"
+    bin/generate_client -l csharp-netcore
+  fi
+
+  docker-compose up -d --no-deps test-dotnet
+  rm -rf ./out/csharp-netcore/src/Org.OpenAPITools.Test/Api
+  rm -rf ./out/csharp-netcore/src/Org.OpenAPITools.Test/Model
+  docker-compose exec -T test-dotnet bash -c "cp -v -R ./test/csharp-netcore/* -t ./out/csharp-netcore/src/Org.OpenAPITools.Test/"
+
+  announce "Running C# Integration Tests"
+  docker-compose exec -T \
+    test-dotnet \
+    bash -c "cd out/csharp-netcore && dotnet test"
+}
+
+run_all_tests(){
+  run_python_tests
+  run_csharp_tests
```

Comment:
The C# tests use the `admin` API key from before the Python tests run, even though the Python test rotate `admin` API key in one of the tests. I think you would just need to reset the env var here:
```suggestion
  run_python_tests
  export CONJUR_AUTHN_API_KEY="$(docker-compose exec conjur conjurctl role retrieve-key dev:user:admin | tr -d '\r')"
  run_csharp_tests
```